testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js

```diff
@@ -13,11 +13,21 @@ const DROP_DOWN_BUTTON_ACTION_CLASS = "dx-dropdownbutton-action";
 const DROP_DOWN_BUTTON_TOGGLE_CLASS = "dx-dropdownbutton-toggle";
 
 QUnit.testStart(() => {
-    const markup = '' +
-        '<div id="container">' +
-            '<div id="dropDownButton"></div>' +
-            '<div id="dropDownButton2"></div>' +
-        '</div>';
+    const markup =
```

Comment:
Can we add these styles via the `css` method in the test case?

Reply:
Yes, thanks